build.sbt

```diff
@@ -1,36 +1,37 @@
 name := "sangria-ion"
 organization := "org.sangria-graphql"
-version := "1.0.1-SNAPSHOT"
+mimaPreviousArtifacts := Set("org.sangria-graphql" %% "sangria-ion" % "1.0.0")
 
 description := "Sangria Amazon Ion marshalling"
 homepage := Some(url("http://sangria-graphql.org"))
 licenses := Seq("Apache License, ASL Version 2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0"))
 
-scalaVersion := "2.12.1"
-crossScalaVersions := Seq("2.11.8", "2.12.1")
+scalaVersion := "2.12.10"
```

Comment:
You don't want to use the `2.13.0` by default?

Reply:
Yes, we should be consistent here—thanks for catching!